addon-resizer/Makefile

```diff
@@ -22,7 +22,7 @@ ARCH ?= amd64
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
 GOARM=7
-GOLANG_VERSION = 1.16
+GOLANG_VERSION = 1.17
 REGISTRY = gcr.io/k8s-staging-autoscaling
```

Comment:
Now that we have forked from the upstream repo, we should change the registry and tag.